src/styles/_variables.scss

```diff
@@ -24,6 +24,8 @@ $md-red-60: #D4371C !default;
 
 $md-black-100: #000000;
 
+$md-yellow-40: #FC9D03 !default;
```

Comment:
yellow-40 is already in this file (line 21), together with the grays. Maybe it was added by a recent merge.